index.html

```diff
@@ -101,18 +102,30 @@ <h2>About</h2>
 		<h2>Contact</h2>
```

Comment:
I think your contact should sit inside your main so that you have:
`header`
`main`
`footer`